runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientFactory.java

```diff
@@ -53,8 +54,13 @@ public KafkaClientFactory(
         final Long2ObjectHashMap<KafkaBindingConfig> bindings = new Long2ObjectHashMap<>();
         final KafkaMergedBudgetAccountant accountant = new KafkaMergedBudgetAccountant(context);
 
-        this.connectionPool = new KafkaClientConnectionPool(
-            config, context, bindings::get, accountant.creditor());
+        this.connectionPool = new KafkaClientConnectionPool(config, context, accountant.creditor());
```

Comment:
Local variable is sufficient, no need for field.

```suggestion
        KafkaClientConnectionPool connectionPool = new KafkaClientConnectionPool(config, context, accountant.creditor());
```